### DISCUSSION TITLE
ultima 7.0.0 can't display left menu

### DISCUSSION THREAD

**Discussion by C0160** [primefaces/community#1243] [PrimeFaces Templates] [answered]
When I use ultima theme 7.0.0 and primefaces 13.0.0 for jakarta in a project, the project can startup on glassfish 7.0.6 ,but not display left menu.When debug , found UltimaMenu.processEvent() method not invoke,seems @ListenerFor not work .Anyone can help me?
pom.xml
 <dependencies>
       <dependency>
            <groupId>jakarta.platform</groupId>
            <artifactId>jakarta.jakartaee-api</artifactId>
            <version>10.0.0</version>
            <scope>provided</scope>
        </dependency>
        <dependency>
            <groupId>org.primefaces</groupId>
            <artifactId>primefaces</artifactId>
            <version>13.0.0</version>
            <classifier>jakarta</classifier>
        </dependency>
        <dependency>
            <groupId>org.glassfish</groupId>
            <artifactId>jakarta.faces</artifactId>
            <version>4.0.2</version>
        </dependency>
        <dependency>
            <groupId>jakarta.faces</groupId>
            <artifactId>jakarta.faces-api</artifactId>
            <version>4.0.1</version>
        </dependency>
        <dependency>
            <groupId>jakarta.el</groupId>
            <artifactId>jakarta.el-api</artifactId>
            <version>5.0.1</version>
            <scope>provided</scope>
        </dependency>
        <dependency>
            <groupId>jakarta.servlet.jsp.jstl</groupId>
            <artifactId>jakarta.servlet.jsp.jstl-api</artifactId>
            <version>3.0.0</version>
        </dependency>
        <dependency>
            <groupId>jakarta.servlet</groupId>
            <artifactId>jakarta.servlet-api</artifactId>
            <version>6.0.0</version>
        </dependency>
……
    </dependencies>

**Comment by melloware** [comment, marked as answer]
Ultima is working for me in PF13 and PF14 fine....

![image](https://github.com/user-attachments/assets/d648c62c-601f-4f4a-bdbf-5c84280af539)
